bundles/org.openhab.binding.ekey/src/main/java/org/openhab/binding/ekey/internal/handler/EkeyHandler.java

```diff
@@ -82,10 +82,10 @@ public void initialize() {
 
             scheduler.submit(() -> {
                 populateChannels(config.protocol);
-                String readerThreadName = "OH-binding-" + getThing().getUID().getAsString();
 
                 EkeyUdpPacketReceiver localReceiver = receiver = new EkeyUdpPacketReceiver(
-                        Objects.requireNonNullElse(config.natIp, config.ipAddress), config.port, readerThreadName);
+                        Objects.requireNonNullElse(config.natIp, config.ipAddress), config.port,
+                        getThing().getUID().getAsString());
```

Comment:
Do you need to call `.getAsString()` here? .. Doesn't `getThing().getUID()` render as a string anyway?

Reply:
Oh sorry. I see my mistake.